DOCS-1957: Update security image robot (#2815)
viamrobotics/docs: docs/tutorials/projects/send-security-photo.md

### Configure the detection camera

To be able to test that the vision service is working, add a [transform camera](/components/camera/transform/) which will add bounding boxes and labels around the objects the service detects.

Click the **+** (Create) button next to your main part in the left-hand menu and select **Component**.

> Start typing "webcam" and select **camera / transform**.

Reviewer: ```suggestion
Start typing "transform" and select **camera / transform**.
```
I think?

### Create the main script file

On your computer, navigate to the directory where you want to put the code for this project. Create a file there called <file>chocolate_security.py</file>. This will be the main script for the machine.

Copy the following code and paste it into <file>chocolate_security.py</file>:

```python
import asyncio
import os
import yagmail

from viam.robot.client import RobotClient
from viam.rpc.dial import Credentials, DialOptions
from viam.services.vision import VisionClient, VisModelConfig, \
    VisModelType, Detection

> # Set environment variables. You can get them from your machine's 'Code sample' tab

Author: ```suggestion
# Set environment variables. You can get them from your machine's CONNECT tab
```

> api_key= os.getenv('API_KEY') or ''

Author: ```suggestion
api_key = os.getenv('API_KEY') or ''
```
